src/App.js

```diff
@@ -100,6 +101,12 @@ class App extends React.Component {
         if (account && this.state.wrapper) {
           this.state.wrapper.setAccounts([account])
         }
+
+        if (account) {
+          getIsContractAccount(getWeb3(web3Providers.wallet))
+            .then(isContractAccount => this.setState({ isContractAccount }))
+            .catch(err => err)
```

Comment:
Unsure if this is the way to do it, but otherwise the linter was complaining

Reply:
Ehhh yes, this is a bit annoying but generally if there's nothing else to do, the warning serves as a reminder to log this error :).